codebuild/spec/buildspec_omnibus.yml

```diff
@@ -29,6 +29,12 @@ batch:
         privileged-mode: true
         compute-type: BUILD_GENERAL1_LARGE
 
+    - identifier: s2nIntegrationAwsLcInSourceBuild
+      buildspec: codebuild/spec/buildspec_ubuntu_integ_awslc_insourcebuild.yml
```

Comment:
Is "Integration" the right name? Usually "integration" indicates that a build runs the integration tests, and if I'm understanding this right it only runs the unit tests.

Maybe just "s2nAwsLcInSourceBuild"?

Reply:
Right, it's not integration tests, will update the name.

Reply:
Loose convention for codebuild job names has been s2n TestType Libcrypto [Batch|Compiler|Coverage], so something along the lines of `s2nUnitAwsLcInSource`, or if nothing else, include the Unit after the s2n.

Reply:
It makes sense to me.